docs/docs/20-quickstart.md

```diff
@@ -146,11 +146,15 @@ At the end of this process:
 * The Argo CD dashboard will be accessible at [localhost:31443](https://localhost:31443).
 
   The username and password are both `admin`.
+  
+  ![Argo-dashboard-screenshot](../static/img/argo-dashboard-1.png)
 
 * The Kargo dashboard will be accessible at [localhost:31444](https://localhost:31444).
 
   The admin password is `admin`.
 
+  ![Kargo-dashboard-screenshot](../static/img/kargo-dashboard-1.png)
```

Comment:
Same as [this comment](https://github.com/akuity/kargo/pull/2521/files#r1791978562)